include/RAJA/config.hpp.in

```diff
@@ -196,34 +196,45 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 #if defined(RAJA_ENABLE_CUDA)
 #define RAJA_ALIGN_DATA(d)
 #else
-
-#if __ICC < 1300  // use alignment intrinsic
 #define RAJA_ALIGN_DATA(d) __assume_aligned(d, DATA_ALIGN)
-#else
-#define RAJA_ALIGN_DATA(d)  // TODO: Define this...
 #endif
 
+#if defined(_OPENMP) && (_OPENMP >= 201307)
+#define RAJA_SIMD  _Pragma("omp simd") \
+                   _Pragma("vector aligned")
+#define RAJA_NoSIMD _Pragma("novector")
+#else
+#define RAJA_SIMD  _Pragma("simd") \
+                   _Pragma("vector aligned")
+#define RAJA_NoSIMD  _Pragma("novector")
 #endif
 
-#define RAJA_SIMD  // TODO: Define this...
-
 
 #elif defined(RAJA_COMPILER_GNU)
 //
 // Configuration options for GNU compilers
 //
 
 #define RAJA_INLINE inline  __attribute__((always_inline))
+#define RAJA_GNU_NoSIMD  __attribute__((noinline, optimize("no-tree-vectorize")))
```

Comment:
`NO_SIMD`

Reply:
Thanks @davidbeckingsale, I've addressed the changes. Regarding Ramesh's work, I had comparable time to solution results without using #pragma forceinline recursive but I can add it in.

Reply:
@artv3 it may be that tests you ran were not as complex as what @Ram3sh was looking at. It may be a good idea to get @Ram3sh code and do some tests.